src/Dfc.CourseDirectory.Web/Controllers/BulkUploadController.cs

```diff
@@ -314,32 +310,41 @@ public async Task<IActionResult> DeleteFile(DeleteFileViewModel model)
                 var archiveFilesResult = _blobService.ArchiveFiles($"{UKPRN.ToString()}/Bulk Upload/Files/");
             }
 
-
             // COUR-1927 move the delete to a background worker because it's timing out for large files.
             bool deleteSuccess = false;
```

Comment:
This is very curious - I don't see how this could've ever been set to true before the method completed?